python/pid_reports/README.md

```diff
@@ -0,0 +1,71 @@
+# PID Reports
+
+## Overview
+
+This folder contains scripts that generate reports on Persistent Identifier (PID) usage in a Dataverse instance. These scripts specifically identify cases where PIDs were not found, which may indicate:
+
+- In-the-wild use of draft PIDs
+- Posting of PIDs with typos
+- PIDs with extra characters (e.g., trailing periods)
+- Other malformed PID references
+
+## Scripts
+
+The main script in this folder is `dcpidreport.py`, which checks DataCite for DOI resolution and generates reports on failures. Anything reported via this script indicates that someone tried to resolve the specified DOI, i.e. via https://doi.org/* . DataCite can sometimes be more than a month delayed in updating it's reports - the script is able to handle this.
+
+A second script, `pidreport.py` performs similar functions for any PIDs. However, it relies on functionality to create an initial PIDFailures report that is not yet merged into the standard Dataverse distribution from https://github.org/IQSS/dataverse.
+The benefit of this message is that the results are available every month and they capture any call to Dataverse requiring a PID (i.e. where someone may have posted a direct, incorrect link to a dataset page, versus DataCite only reporting DOI resolution failures).
+
+## Purpose
+
+These reports help maintain the integrity of your Dataverse's persistent identifier system by:
+- Identifying problematic PID references
+- Alerting administrators to potential issues
+- Providing data for troubleshooting and correction
+
+## Usage
+
+The scripts are designed to be run periodically (typically monthly) via a cron job.
+
+### Configuration
+
+Before using these scripts, you need to configure several variables in each script:
```

Comment:
An obvious future enhancement would be to extract these settings into a config file.